.flake8

```diff
@@ -0,0 +1,15 @@
+[flake8]
+ignore=
+  E127,  # over indented
+  E128,  # under indented
+  E302,  # blank lines
+  E303,  # blank lines
```

Comment:
Just a curious question following my latest PR...Why do we ignore these line linting errors? I imagine that this one is a very easy one to fix while making the code look much better and pep8 complaint...

Reply:
because I'm going to soon turn on the linter on PRs and I need all my existing code to pass first.  There's a lot that won't pass right now, so just getting a minimum number of tests in -- already has taken 6 hours to get the code up to that level.